Nos/Views/NoteCard.swift

```diff
@@ -42,9 +42,7 @@ struct NoteCard: View {
         if replyCount == 0 {
             return nil
         }
-        let replyCount = replyCount
-        let localized = replyCount == 1 ? Localized.Reply.one : Localized.Reply.many
```

Comment:
This logic did not work for locales that support different pluralization rules than just `one` and `many`.